src/algebra/group/pi.lean

```diff
@@ -43,9 +43,10 @@ instance monoid [∀ i, monoid $ f i] : monoid (Π i : I, f i) :=
 by refine_struct { one := (1 : Π i, f i), mul := (*), npow := λ n x i, (x i) ^ n };
 tactic.pi_instance_derive_field
 
--- the attributes are intentionally out of order. `smul_apply` proves `nsmul_apply`.
-@[to_additive, simp]
```

Comment:
Removing `to_additive` here is not the right fix.

Reply:
I have reverted this change

Reply:
After moving `has_pow` and `has_smul` to `data.pi.algebra`, this `pow_apply` lemma is duplicated